contrib/inventory/foreman.py

```diff
@@ -140,12 +160,14 @@ def _get_session(self):
             self.session.verify = self.foreman_ssl_verify
         return self.session
 
-    def _get_json(self, url, ignore_errors=None):
+    def _get_json(self, url, ignore_errors=None, params={}):
```

Comment:
It's good to know that the default params here is stored in the function defaults. In this case it will work as you expect because you set the values explicitly before using them, but it's good to know there may be unexpected consequences. See http://www.effbot.org/zone/default-values.htm for a full explanation why mutable function defaults are a bad pattern in Python.